test/server/cards/01_SOR/units/LeiaOrganaDefiantPrincess.spec.js

```diff
@@ -0,0 +1,56 @@
+describe('Leia Organa, Defiant Princess', function() {
+    integration(function() {
+        describe('Leia Organa, Defiant Princess\'s when played ability', function() {
+            beforeEach(function () {
+                this.setupTest({
+                    phase: 'action',
+                    player1: {
+                        hand: ['leia-organa#defiant-princess'],
+                        groundArena: ['atst'],
+                        spaceArena: [{ card: 'cartel-spacer', exhausted: false }],
+                        leader: { card: 'jyn-erso#resisting-oppression', deployed: true, exhausted: false }
+                    },
+                    player2: {
+                        groundArena: [{ card: 'wampa', exhausted: false }],
+                        spaceArena: ['alliance-xwing']
+                    }
+                });
+            });
+
+            it('should be able to ready a friendly resource', function () {
```

Comment:
Sorry, I somehow completely missed this until now - technically per RAW Leia should be able to ready a resource for the opponent.

That said, it may be the case we decide there's no reason to ever do that currently so we just shortcut and always do a friendly resource. If you prefer that option lmk and we can confirm with the rest of the team that they think it's a good idea.

Reply:
I think I recall someone telling me the opponent's resource case wasn't actually needed. I'd agree with that too: I can't think of even an edge case where someone would want to ready an opponent's resource(at least in a standard 2 player game).

Reply:
Yeah I vaguely remembered some of the TWI cards keying off of ready resources but none of them would change that. I'm fine to leave as is until some real situation arises where it would be useful